src/literature/components/ReferenceItem/ReferenceItem.scss

```diff
@@ -1,5 +1,4 @@
 .bulleted-inline-list {
-  display: inline-block;
```

Comment:
I think name of the class should be changed, since it's not `inline` anymore

Reply:
I changed it to ```bulleted-list```

Reply:
actually isn't  list bulleted by default unless it is inline? so you can perhaps remove the class.

Reply:
If I don't apply this class, the list items actually lose the bullets
![screenshot from 2018-10-17 09-59-47](https://user-images.githubusercontent.com/11242410/47071370-ac227e80-d1f3-11e8-9ecc-93ad2e1b38ff.png)
That's because of the class ```dib``` on each ```<li>```